core/src/main/java/hudson/WebAppMain.java

```diff
@@ -235,23 +234,6 @@ public Locale get() {
                 throw new NoTempDir(e);
             }
 
-            // Tomcat breaks XSLT with JDK 5.0 and onward. Check if that's the case, and if so,
-            // try to correct it
-            try {
-                TransformerFactory.newInstance();
-                // if this works we are all happy
-            } catch (TransformerFactoryConfigurationError x) {
-                // no it didn't.
-                LOGGER.log(WARNING, "XSLT not configured correctly. Hudson will try to fix this. See https://bz.apache.org/bugzilla/show_bug.cgi?id=40895 for more details",x);
-                System.setProperty(TransformerFactory.class.getName(),"com.sun.org.apache.xalan.internal.xsltc.trax.TransformerFactoryImpl");
-                try {
-                    TransformerFactory.newInstance();
-                    LOGGER.info("XSLT is set to the JAXP RI in JRE");
-                } catch(TransformerFactoryConfigurationError y) {
-                    LOGGER.log(SEVERE, "Failed to correct the problem.");
-                }
-            }
-
```

Comment:
Was triggering `XXE_DTD_TRANSFORM_FACTORY` and `XXE_XSLT_TRANSFORM_FACTORY`, but the `TransformerFactory` is immediately thrown away rather than used for anything, so this is not actually a problem. Furthermore, this entire block of code is a workaround for [a bug that was fixed in Tomcat 5.5](https://bz.apache.org/bugzilla/show_bug.cgi?id=40895), which was [EOLed in 2012](https://tomcat.apache.org/tomcat-55-eol.html), so it's safe to say we don't need it anymore.